libwasmvm/src/cache.rs

```diff
@@ -685,7 +729,7 @@ mod tests {
                 misses: 0,
                 elements_pinned_memory_cache: 1,
                 elements_memory_cache: 0,
-                size_pinned_memory_cache: 3432787,
+                size_pinned_memory_cache: 4956712,
```

Comment:
Do you have the reason this size need to be change to more big?

Reply:
This value increased also in upstream. But in upstream, it is a little bigger than ours. This difference comes from cosmwasm-vm.